extensions/mongo-client/deployment/pom.xml

```
@@ -0,0 +1,61 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>io.quarkus</groupId>
+        <artifactId>quarkus-mongodb-client-parent</artifactId>
+        <version>999-SNAPSHOT</version>
+    </parent>
+
+    <artifactId>quarkus-mongodb-client-deployment</artifactId>
+    <name>Quarkus - Mongo - Client - Deployment</name>
```

```suggestion
    <name>Quarkus - MongoDB Client - Deployment</name>
```
would be better, I think.

---

I changed the artifactId, forgot about the name. So definitely!